### PR TITLE
Add Homebox to Inventory Management apps

### PR DESCRIPTION
**Pull request by Kovah**
Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] If you are adding software forked from another active project, please provide/link to a clear list of differences between both.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

Superseeds #188 which was closed without comments or details, with the original fork being deleted already.

### CODE REVIEW

File: software/homebox-inventory.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: "Homebox"
+website_url: "https://hay-kot.github.io/homebox/"
+source_code_url: "https://github.com/hay-kot/homebox"
+description: "An inventory and organization system built for the Home User with a focus on simplicity and ease of use."
```
Comment (Ki-er) [inline software/homebox-inventory.yml:4]:
```suggestion
description: "An inventory and organization system built for the home user with a focus on simplicity and ease of use."
```

**Review by Ki-er — CHANGES_REQUESTED** [review]
Tiny tweak to fix sentence case.

**Review by Ki-er — APPROVED** [review]
LGTM- Thanks

**Review by nodiscc — COMMENTED** [review]
There is another project named `Homebox` (https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/software/homebox.yml). I think nothing technically prevents 2 projects from having the same `name:`, but shouldn't we rename this one to `Homebox Inventory` just to be safe?

**Comment by Ki-er** [comment]
Homebox inventory would make sense as when running the actions etc, the distinction there too would be useful.

File: software/homebox-inventory.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+name: "Homebox"
```
Comment (kokomo123) [inline software/homebox-inventory.yml:1]:
```suggestion
name: "Homebox Inventory"
```

**Pull request merged**